release/tune_tests/cloud_tests/workloads/run_cloud_test.py

```diff
@@ -320,7 +320,7 @@ def run_tune_script_for_time(
         # Wait until indicator file exists
         wait_for_run_or_raise(process, indicator_file=indicator_file, timeout=30)
         # Stop experiment (with checkpoint) after some time
-        send_signal_after_wait(process, signal=signal.SIGINT, wait=run_time)
+        send_signal_after_wait(process, signal=signal.SIGUSR1, wait=run_time)
```

Comment:
I wonder if this could make Tuner code simpler, if we just ask user to terminate using this signal? How would a user issue SIGUSR1?

Reply:
I see, this makes sense. I will update Tuner then.